packages/picasso.js/src/web/renderer/svg-renderer/svg-nodes.js

```diff
@@ -45,10 +45,12 @@ const maintainer = (element, item) => {
         val = item.attrs[attr];
       }
       element.setAttribute('dy', val + dy);
-    } else if (item.type === 'text' && attr === 'title' && item.attrs.title) {
-      const t = element.ownerDocument.createElementNS(svgNs, 'title');
-      t.textContent = item.attrs.title;
-      element.appendChild(t);
+    } else if (item.type === 'text' && attr === 'title') {
+      if (item.attrs.title) {
```

Comment:
do proper type check to support falsy values

Reply:
`item.attrs.title` should always be a string - I don't see the point in supporting an empty value here?

Reply:
I think you're putting a bit too much responsibility on the user.  The primary case would be to allow 0 as a number.